src/hatti/views/table.cljs

```diff
@@ -95,7 +95,17 @@
        :rowHeight 24
        :enableTextSelectionOnCells true})
 
-(defn sg-init [data form is-filtered-dataview?]
+(defn bind-external-slick-grid-event-handlers
```

Comment:
Any ideas on a testing strategy for this, that doesn't involve introspection into the `grid`, @pld?